README.md

```diff
@@ -9,8 +9,8 @@ We welcome and appreciate all contributions._
 
 # VCert
 
-[![GoDoc](https://godoc.org/github.com/Venafi/vcert?status.svg)](https://godoc.org/github.com/Venafi/vcert)  [![Go Report Card](https://goreportcard.com/badge/github.com/Venafi/vcert)](https://goreportcard.com/report/github.com/Venafi/vcert)
-[![Used By](https://sourcegraph.com/github.com/Venafi/vcert/-/badge.svg)](https://sourcegraph.com/github.com/Venafi/vcert?badge)
+[![GoDoc](https://godoc.org/github.com/Venafi/vcert/v4?status.svg)](https://godoc.org/github.com/Venafi/vcert/v4)  [![Go Report Card](https://goreportcard.com/badge/github.com/Venafi/vcert/v4)](https://goreportcard.com/report/github.com/Venafi/vcert/v4)
+[![Used By](https://sourcegraph.com/github.com/Venafi/vcert/v4/-/badge.svg)](https://sourcegraph.com/github.com/Venafi/vcert/v4?badge)
```

Comment:
I believe the sourcegraph url is based on the repo url so v4 should not be part of that